structure/0006-governance.md

```diff
@@ -2,53 +2,152 @@
+### Team Members
 
-### Removal of Staff
-Any team member (up to and including administrative board members) may be removed by a majority vote of all staff members above and below.
- * For members of the administrative board, this would be all staff members
- * For technical leads, this would be all technical staff members, as well as all members of the administrative board
- * For the QSL Module A lead, this would be all team members of QSL Module A, all technical leads, and all members of the administrative board
- * For a community moderator, this would be all community moderators and all members of the administrative board
+A team member is a member of the organization assigned to a specific team on GitHub.
+A member of the organization may be assigned to multiple teams.
+A team must have at least one member.
+
+### Team Leads
+
+A team lead is a team member with the [Team Maintainer](https://docs.github.com/en/organizations/organizing-members-into-teams/assigning-the-team-maintainer-role-to-a-team-member#about-team-maintainers) 
+role for the relevant team on GitHub.
+A member of the organization may be a team lead for multiple teams.
+Each team must have at least one team lead.
+A team lead may be nominated via general consensus of the team, or by an election, the process being detailed below.
+
+### Team Responsibilities
+
+Team responsibilities will vary and be detailed on a team-by-team basis.
+In general, a team is responsible for performing the tasks outlined in the RFC that proposed its creation.
+
+Most teams will also maintain one or more repositories on GitHub.
+The team is responsible for reviewing pull requests and ensuring that they meet The Quilt Project's goals in quality and purpose.
+Once a team has decided that a pull request is suitable for merging, a team lead will be responsible for the actual merging. Team leads can also decide to allow some or all team members to perform merges.
+
+### Creating a Team
+
+A new team can be created by passing an RFC proposing that team.
+The initial team leads must be specified in the pull request adding the RFC and must approve the RFC before it is merged.
+Once the [RFC process](0001-rfc-process.md) is concluded and the RFC us merged,
+a new team with the specified leads will be created on GitHub by the Administrative Board.
+
+The RFC proposing the new team should contain the following major sections:
+
+- **Motivation**:
+  - Why is this team needed?
+  - What problems will this team solve?
+  - What tasks will this team be responsible for?
+
+- **Explanation**:
+  - What will this team *do*?
+  - Who will be the team leads?
```

Comment:
Does this make sense to hardcode into an RFC? I feel like team composition shouldn't require RFC changes.

Reply:
I think the intent here is to have a ground truth about team composition somewhere.
That being said, it does mention a bit earlier that initial leads should be specified in the PR (rather than the RFC), whch somewhat contradicts this sentiment.

Reply:
The team composition should only be in the PR, I forgot to remove that line